curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/review-algorithmic-thinking-by-building-a-dice-game/657caa69db80ef25862b1b17.md

```diff
@@ -1,15 +1,13 @@
 ---
 id: 657caa69db80ef25862b1b17
-title: Step 24
+title: Step 6
 challengeType: 0
-dashedName: step-24
+dashedName: step-6
```

Comment:
the editable region should include the `rollDiceBtn.addEventListener` code since updates are supposed to the made there